slices/node-acorn.yaml

```
@@ -0,0 +1,35 @@
+package: node-acorn
+
+slices:
+  libs:
+    essential:
+      - node-xtend_libs
+    contents:
+      # basically avoiding all the .map files (line tables) and unittest drivers
+      /usr/share/nodejs/acorn/**:
```

```suggestion
      # do you need /usr/share/nodejs/acorn/dist/bin.js? If not, the following pattern needs to be fine tuned
      /usr/share/nodejs/acorn/dist/*:
      /usr/share/nodejs/acorn/package.json:
```

This glob is bringing in `/usr/share/nodejs/acorn/bin/acorn`, which I guess you don't want? At least not in this slice.

---

Done

---

What was done? I don't see any diffs